src/tensorlake/function_executor/proto/function_executor.proto

```diff
@@ -125,17 +153,15 @@ enum TaskFailureReason {
 
 message RunTaskResponse {
     optional string task_id = 1;
-    repeated SerializedObject function_outputs = 2;
+    repeated SerializedObjectInsideBLOB function_outputs = 2;
     // Edges that the function wants the invocation to be routed to.
     repeated string next_functions = 3;
-    optional string stdout = 4;
-    optional string stderr = 5;
-    optional bool is_reducer = 6;
-    optional Metrics metrics = 7;
-    optional TaskOutcomeCode outcome_code = 8;
-    optional TaskFailureReason failure_reason = 9;
+    optional bool is_reducer = 4;
+    optional Metrics metrics = 5;
+    optional TaskOutcomeCode outcome_code = 6;
+    optional TaskFailureReason failure_reason = 7;
     // User payload for invocation error if task failed with invocation error.
-    optional SerializedObject invocation_error_output = 10;
+    optional SerializedObjectInsideBLOB invocation_error_output = 8;
```

Comment:
Not a comment but just a clarification. We are storing the error from the functions inside the URL for outputs? Makes sense to do it that way

Reply:
It's an S3 object, this allows to remove limits on InvocationError.message size and let users dump any json they want there. Server already stores this as DataPayload. Might not be what you expected but this is flexible and works. Good enough for now imo.

Reply:
These InvocationError.message strings are also customer data and we handle it consistently (in S3 blobs). I think this is more important for long term to be consistent.

Reply:
Also answering your question, I guess I see what you meant.
Yes, we store InvocationError.message in the BLOB used for function outputs.